jetbrains-core/src/software/aws/toolkits/jetbrains/services/telemetry/TelemetryService.kt

```diff
@@ -113,7 +85,9 @@ class DefaultTelemetryService(settings: AwsSettings) :
         TelemetryService.subscribe(this)
         TelemetryService.syncPublisher().notify(settings.isTelemetryEnabled)
 
-        record("ToolkitStart").also {
+        // The auto generated telemetry cannot be used here. It tries to get the instance while
+        // constructing it which causes a circular dependency issue.
+        record("session_start").also {
```

Comment:
We may be able to move this to a postStartupActivity to fix that

Reply:
Nvm, that looks to be project level. Application level isn't do-able until 193 I think

Reply:
Will add TODO to fix it when min is 193